src/main/java/seedu/address/logic/Messages.java

```diff
@@ -48,4 +48,17 @@ public static String format(Person person) {
         return builder.toString();
     }
 
+    /**
+     * Formats the {@code people} for display to the user.
+     */
+    public static String format(Person[] people) {
+        int size = people.length;
+        final StringBuilder builder = new StringBuilder();
+        for (int i = 0; i < size - 1; i++) {
+            builder.append(format(people[i]))
```

Comment:
Could be too long if many people are deleted (person.toString() quite long :0), but I think this is good too!

Reply:
that is true :P I will work on this output once the details of the fosterers are settled!